patroni/postgresql.py

```diff
@@ -964,3 +971,37 @@ def basebackup(self, conn_url, env):
                 time.sleep(5)
 
         return ret
+
+    @staticmethod
+    def postgres_version_to_int(pg_version):
+        """ Convert the server_version to integer
+
+        >>> Postgresql.postgres_version_to_int('9.5.3')
+        90503
+        >>> Postgresql.postgres_version_to_int('9.3.13')
+        90313
+        >>> Postgresql.postgres_version_to_int('10.1')
+        100100
```

Comment:
I believe this would become 100001, see:
http://www.databasesoup.com/2016/05/changing-postgresql-version-numbering.html